audbcards/core/datacard.py

```diff
@@ -123,6 +127,52 @@ def example_media(self) -> typing.Optional[str]:
             media = None
         return media
 
+    @property
+    def file_duration_distribution(self) -> str:
+        r"""Min/max of files durations, and plotted distribution.
+
+        This generates a single line
+        containing the min/max values
+        of files durations
+        and an inline plot of the corresponding distribution,
+        e.g. 1 s |distribution| 10 s.
+        If used outside of ``sphinx``,
+        it return a string with the minimum and maximum values,
+        e.g. 1 s .. 10 s.
+
+        .. |distribution| image:: docs/images/file-duration-distribution.png
+
+        """
+        min_ = 0
+        max_ = 0
+        unit = 's'
+        durations = self.dataset.file_durations
```

Comment:
Afaics, the durations will rely on `audb.dependencies( )`  (`media` and `durations`).  Is there a situation conceivable where a database has no media? For example, one might try to (ab)use audb to represent a text copus that per defintion would have no media - that would then return an empty list.

Reply:
In the tests we have `bare_db`, which is a database without any files.

Reply:
But I think that database is only used in a test that tries not to plot the image, so maybe we also need to add an `if` statement around `_plot_distribution()`

Reply:
The plotting test was not using `basre_db`. I extended the test by it, and fixed the plotting code to handle that case as well.

Reply:
I have checked the resulting image file. When not input media, the image is empty. when using `medium_db` I am seing a uniform distribution.

So this works, I will resolve this conversation.